recipes-core/systemd/systemd_%.bbappend

```diff
@@ -22,6 +15,7 @@ FILES_${PN} += " \
 
 PACKAGECONFIG_remove = " \
     networkd \
+    serial-getty-generator \
```

Comment:
> systemd has a native feature that allows generation and instantiatiation
> of serial-getty@.service instances for virtualizer consoles.

Hum, I do not understand. This line makes it so that it is not used?

Reply:
Or this is two changes in one? One that remove our own `ln -s` for getty@.service and relies on `systemd-getty-generator`. And the PACKAGECONFIG_removes generation of serial gettys... ?

Reply:
`serial-getty-generator` PACKAGECONFIG is disabled by default? From systemd recipe:

```
# Use the upstream systemd serial-getty@.service and rely on
# systemd-getty-generator instead of using the OE-core specific
# systemd-serialgetty.bb - not enabled by default.
PACKAGECONFIG[serial-getty-generator] = ""
```